.github/workflows/release.yml

```
@@ -42,32 +41,33 @@ jobs:
-      - name: Get Last Merged PR
-        id: get_merged_pr
-        uses: actions-ecosystem/action-get-merged-pull-request@v1
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
         
-      # Determine the release type (major, minor, patch) based on Last Merged PR Title  
-      - name: Determine Release Type from PR Title
+      # Determine the release type (major, minor, patch) based on commit messages  
+      - name: Determine Release Type
         id: determine_release
         run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)